tests/test_livepatch.py

```diff
@@ -42,6 +43,15 @@ def test_enable_livepatch_invalid_token(self):
         self.assertEqual(3, process.returncode)
         self.assertIn('Invalid or missing Livepatch token', process.stderr)
 
+    def test_enable_livepatch_unsupported_on_non_x86_64(self):
+        """Livepatch is only supported on x86_64."""
```

Comment:
A more appropriate test name and docstring would be that livepatch is not supported on i686.

Reply:
fixed